### DISCUSSION TITLE
Happy New Year 2022!!!!. graphql-ez is ready to use in production?

### DISCUSSION THREAD

**Discussion by nghiepdev** [PabloSzx/graphql-ez#709] [General]
Hey,
I'm working on `mercurius-js` and looking for an alternative.
I've known `graphql-ez` for a while. But I'm not sure `graphql-ez` is ready to use in production?

Thanks!

**Comment by PabloSzx** [comment]
Hi, it's 100% production ready

**Reply by PabloSzx** [reply]
If you find any issues feel free to report it 👍I'm actively maintaining and using this project

**Reply by nghiepdev** [reply]
Thank you @PabloSzx 
Because I don't see `graphql-ez` in the list of products by The Guild
.
